### ISSUE TITLE
Change in WGAN-GP Discriminator Loss

### ISSUE THREAD

**Issue by Alikerin** [sdv-dev/CTGAN#82]
Please I will like to know the reason why the original WGAN-GP loss was modified from `loss_d = -(torch.mean(y_real) + torch.mean(y_fake))` to `loss_d = -(torch.mean(y_real) - torch.mean(y_fake))` in your code. Why did you change the sign? Thank you.

**Comment by Alikerin** [comment]
Sorry, my bad!
